system-contracts/contracts/Constants.sol

```diff
@@ -126,3 +133,7 @@ uint256 constant STATE_DIFF_DERIVED_KEY_OFFSET = 52;
 uint256 constant STATE_DIFF_ENUM_INDEX_OFFSET = 84;
 /// @dev The offset of the final value in a storage diff.
 uint256 constant STATE_DIFF_FINAL_VALUE_OFFSET = 124;
+
+/// @dev Total number of bytes in a blob. Blob = 4096 field elements * 31 bytes per field element
+/// @dev 4844 defines it as 131_072 but we use 4096 * 31 within our circuits to always fit within a field element
+uint256 constant BLOB_SIZE_BYTES = 126_976;
```

Comment:
Is this format Ethereum compatible? 

From the spec I see `BYTES_PER_FIELD_ELEMENT`, so just curious whether we commit to the data correct

Reply:
I found it! In the circuit, we have a blob hash that is different from the Ethereum format but circuit proves that both commitments contains the same data. Maybe worth to make a comment for that.